### DISCUSSION TITLE
bracket auto completion doesn't work

### DISCUSSION THREAD

**Discussion by root357** [cpeditor/cpeditor#1298] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Auto complete doesn't work. When I enter a left bracket in cpeditor, there should be a right bracket after it, but there isn't.

### Expected Behavior

Let's say `I` is the cursor, I entered the left bracket like `aaa(Ibbb`. It should be `aaa(I)bbb` after that, but the right bracket didn't appear.

### Steps To Reproduce

1. Opens a file 2. Type some code, and enter a left bracket. 3. There isn't a right bracket after the left one.

### CP Editor Version

6.10.3

### OS

lUbuntu 23.10

### Are you willing to contribute to this?

- [ ] I'm ready to fix this (and will open a PR soon)
- [ ] I'd like to have a try (with the help of the maintainers)
- [X] No, thanks

**Comment by ouuan** [comment]
Please check your parenthesis auto-complete settings.

**Reply by root357** [reply]
I've checked the settings but it seems that parenthesis auto-completion is enabled.
[Screenshot 1](https://cdn.luogu.com.cn/upload/image_hosting/fmuw705c.png)
[Screenshot 2](https://cdn.luogu.com.cn/upload/image_hosting/d502q1i1.png)

**Reply by ouuan** [reply]
Check the setting for `()`. It's `''` in your screenshot.

**Reply by root357** [reply]
well it's enabled too.

**Comment by ouuan** [comment]
Emm, cannot reproduce 🤔

**Comment by root357** [comment]
hmmm I've tried reinstalling the app and installing a older version, but it did not solve the problem. Maybe I should try using this app on other systems?

发自我的小米
在 Yufan You ***@***.***>，2024年2月4日 12:42写道：

Emm, cannot reproduce 🤔
